go.mod

```diff
@@ -71,7 +71,7 @@ require (
 )
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v1.4.0-beta.2.0.20200728183644-eb6354a11860
+	github.com/containerd/containerd => github.com/containerd/containerd v1.4.0-beta.2.0.20200730150746-fa1220fce33f
```

Comment:
Can we just vendor rc.1 ?

Reply:
No objections for vendoring newer containerd. Was just trying to keep changes in this PR minimal and specific to the auth feature.